packages/slice-machine/lib/models/common/Slice.ts

```diff
@@ -14,3 +14,7 @@ export interface SliceSaveBody extends SliceBody {
 export interface SliceCreateResponse {
   variationId: string;
 }
+
+export type DeleteSliceResponse =
+  | { err: unknown; reason: string; status: number; type: "error" | "warning" }
+  | Record<string, never>;
```

Comment:
What is this type for?

Reply:
Are you referring to line 20? It's the success response which is just `{}`

If you mean the whole type, it will be used in the saga action response

Reply:
Yep was asking about line 20.